bibliopixel/main/demo.py

```diff
@@ -0,0 +1,60 @@
+import webbrowser
+
+from bibliopixel.drivers.SimPixel import DriverSimPixel
+from bibliopixel import LEDCircle, LEDMatrix, layout
+
+SIMPIXEL_URL = 'http://beta.simpixel.io'
+IMPORT_ERROR_TEXT = """
+Please install the BiblioPixelAnimations library from here:
+
+https://github.com/ManiacalLabs/BiblioPixelAnimations
+
+"""
+
+
+def bloom(args):
+    driver = DriverSimPixel(args.width * args.height)
+    led = LEDMatrix(driver, width=args.width, height=args.height)
+
+    from BiblioPixelAnimations.matrix.bloom import Bloom
+    return Bloom(led)
+
```

Comment:
What exactly is wrong with this? You said you had issues?? What does it do instead?